languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/added.txt

```diff
@@ -5,4 +5,83 @@
 # IMPORTANT: always use only straight (typewriter) apostrophes, not typographical apostrophes
 # Apostrophes in words trigger specific tokenization: common forms (e.g., L'Obs) are multitoken, while non regular usage (e.g., Crit'Air) remain unsplit.
 #separatorRegExp=;
-Quick;Quick;Z m sp
\ No newline at end of file
+Quick;Quick;Z m sp
+Spontex;Spontex;Z e sp;
+Sganarelle;Sganarelle;Z m s
+antimétabole;antimétabole;Z f s
+antimétaboles;antimétabole;Z f p
+Ennius;Ennius;Z m s
+Ouigo;Ouigo;Z m sp
+InOui;InOui;Z m sp
+Robert Badinter;Robert Badinter;Z m s
+ultratransformé;ultratransformé;J m s
+ultratransformée;ultratransformé;J f s
+ultratransformés;ultratransformé;J m p
+ultratransformées;ultratransformé;J f p
+diglycéride;diglycéride;N m s
+diglycérides;diglycéride;N m p
+monoglycéride;monoglycéride;N m s
+monoglycérides;monoglycéride;N m p
+tapinose;tapinose;N f s
+tapinoses;tapinose;N f p
+Pangloss;Pangloss;Z m s
+Toinette;Toinette;Z f s
+Diafoirus;Diafoirus;Z m s
+Mercutio;Mercutio;Z m s
+Tybalt;Tybalt;Z m s
+Missak Manouchian;Missak Manouchian;Z m s
+Mélinée Manouchian;Mélinée Manouchian;Z f s
```

Comment:
This won't work here. Add multi-token expressions to multiwords.txt

Reply:
Thanks Jaume! Just fixed it, merging.

Reply:
Dear lord, thanks, it was definitely too late to work :/